stac_validator/stac_validator.py

```diff
@@ -180,7 +153,15 @@ def validate_json(self, stac_content, schema):
             logger.exception("STAC error")
             return False, f"{error}"
 
-    def _update_status(self, old_status, new_status):
+    @staticmethod
+    def _update_status(old_status, new_status):
```

Comment:
A question: This is not the first place I've seen a method name prepended with an underscore. Can you speak to the convention and what it denotes?

Reply:
As I understand it, its simply meant to show the method should be considered private (even though it is actually public).